src/lazy/value.rs

```diff
@@ -257,7 +257,7 @@ impl<'top, D: Decoder> LazyValue<'top, D> {
         }
     }
 
-    pub fn to_owned(&self) -> LazyElement<D> {
```

Comment:
🪧 Clippy complains that methods named `to_owned` usually take `self` by value. `LazyValue` implements `Copy`, so we can take `self` by value instead of by reference.